setup.py

```diff
@@ -14,7 +14,7 @@
     "aiohttp<4",
     "aiohttp_swagger<2",
     "Jinja2>=3",
-    "fsspec>=2021.7.0",
+    "fsspec>=2021.7.0,<2023.1",
```

Comment:
to fix errors like https://github.com/iterative/mlem/actions/runs/4022310524/jobs/6911940431

didn't have time to investigate

Reply:
Create an issue?